libkbfs/folder_block_ops.go

```diff
@@ -60,6 +66,8 @@ func (si *syncInfo) DeepCopy(codec Codec) (*syncInfo, error) {
 			return nil, err
 		}
 	}
+	newSi.toCleanIfUnused = make([]mdToCleanIfUnused, len(si.toCleanIfUnused))
+	copy(newSi.toCleanIfUnused, si.toCleanIfUnused)
```

Comment:
probably worth a comment that we're treating the mds in `toCleanIfUnused` as const, so no need to copy them. Same for the bps, right?